lifecycle_test.go

```diff
@@ -456,6 +456,83 @@ func TestMergePortBindings(t *testing.T) {
 	}
 }
 
+func TestPortMappingCheck(t *testing.T) {
+	makePortMap := func(ports []string) nat.PortMap {
+		out := make(nat.PortMap)
+		for _, port := range ports {
+			// We don't care about the actual binding in this test
+			out[nat.Port(port)] = make([]nat.PortBinding, 0)
+		}
+		return out
+	}
+
+	tests := []struct {
+		name                  string
+		exposedAndMappedPorts nat.PortMap
+		exposedPorts          []string
+		expectError           bool
+	}{
+		{
+			name:                  "No protocol given",
+			exposedAndMappedPorts: makePortMap([]string{"1024/tcp"}),
```

Comment:
suggestion: with the variadic this just becomes the following, more below:
```suggestion
			exposedAndMappedPorts: makePortMap("1024/tcp"),
```

Reply:
https://github.com/testcontainers/testcontainers-go/pull/2658/commits/5fe2f77b1c6101b2c550f182ec4021d828ff5bd0